package.json

```diff
@@ -24,7 +24,7 @@
     "reconnecting-websocket": "^4.4.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
This line is for me to run the React app. Note to self: remember to delete again.